tests/profiles-enabled/Telemetry/conftest.py

```diff
@@ -0,0 +1,99 @@
+import json
```

Comment:
Please rename the ` tests/profiles-enabled/Telemetry` directory to have consistent casing with the rest of our unit tests directory structure